h2o-py/tests/testdir_jira/pyunit_sw_602.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
The formatting of this file seems off. Is it only because of github?

Reply:
good point, let me fix it